pkg/util/kubernetes/helpers.go

```diff
@@ -8,6 +8,10 @@ package kubernetes
 import (
 	"strconv"
 	"strings"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/DataDog/datadog-agent/pkg/util/pointer"
```

Comment:
You may need to add a build flag to avoid pulling a `k8s.io` in the Agent binaries

Reply:
Separated this out into another file for now with `kubeapiserver` and `kubelet` build flags as I wasn't sure if the build flags can be added to this file directly (but if I misunderstood let me know and I can change it!)